DataFormats/ProtonReco/interface/ProtonTrack.h

```diff
@@ -21,22 +21,33 @@ namespace reco
     public:
       /// Default constructor
       ProtonTrack();
-      /// Constructor from refit parameters, fitted vertex and direction, and longitudinal fractional momentum loss
-      ProtonTrack( double chi2, double ndof, const Point& vtx, const Vector& dir, float xi, const CovarianceMatrix& cov = CovarianceMatrix() );
+
+      /// Constructor from refit parameters, fitted vertex and momentum, and longitudinal fractional momentum loss
+      ProtonTrack( double chi2, double ndof, const Point& vtx, const Vector& momentum, float xi, const CovarianceMatrix& cov = CovarianceMatrix() );
 
       /// Indices to the covariance matrix
+      // TODO: what is num_indeces, where used ??
+      // TODO: add (formally) also vtx_x ?? the base class defines dimension = 5
       enum struct Index : unsigned short { xi, th_x, th_y, vtx_y, num_indices };
 
       /// Longitudinal fractional momentum loss
       float xi() const { return xi_; }
+
       /// Absolute uncertainty on longitudinal fractional momentum loss
+      // TODO: rename error -> uncertainty ?
```

Comment:
Actually it seems that there is even a mixed nomenclature within `CMSSW`, with a large dominance of `xxxError` (see e.g. [`reco::Muons`](https://github.com/cms-sw/cmssw/blob/master/DataFormats/MuonReco/interface/Muon.h#L377-L382) (`xxxErr`), [`reco::GsfTrack`](https://github.com/cms-sw/cmssw/blob/master/DataFormats/GsfTrackReco/interface/GsfTrack.h#L78-L94), [`reco::RecoCandidate`](https://github.com/cms-sw/cmssw/blob/master/DataFormats/RecoCandidate/interface/RecoCandidate.h#L60-L63), [`reco::Candidate`](https://github.com/cms-sw/cmssw/blob/master/DataFormats/Candidate/interface/Candidate.h#L249-L252), [`reco::Vertex`](https://github.com/cms-sw/cmssw/blob/master/DataFormats/VertexReco/interface/Vertex.h#L118-L125), and more meaningfully [`reco::Track`](https://github.com/cms-sw/cmssw/blob/master/DataFormats/TrackReco/interface/TrackBase.h#L295-L329) (`xxxError`)). This somehow reminds me of [this](https://xkcd.com/927/) :-)